app/src/main/java/com/example/salonon/API.java

```diff
@@ -40,7 +40,7 @@ public boolean createNewProfile(Profile profile, String password) {
             parameters.put("salonRate", "0.0");
 
             //request code
-            String response = network.post(network.herokuURL + "createuser", parameters);
+            String response = network.post(network.herokuURL + "createuser", parameters, false);
```

Comment:
Pass in a boolean indicating whether you want to use the body or not.  If not, it uses parameters.